Makefile

```diff
@@ -24,4 +26,12 @@ vet:	### Run go vet against code
 	go vet ./...
 
 test: ### Runs application's tests in verbose mode
-	go test -v ./pkg/...
+	go test -v ./...
+
+docker-build:
+	docker build -t $(IMG):$(TAG) .
+
+docker-push:
+	docker push $(IMG):$(TAG)
+
+publish: docker-build docker-push
```

Comment:
what is the purpose of this? are you using it on your side?

Reply:
Yes, but I can remove it.

Reply:
if you use it, you can leave it. just a note, this isn't something that you should be doing all the time, we don't want to push everything, you can test controllers by running it locally, instead of in the cluster